Sources/SourceKitLSP/TestDiscovery.swift

```diff
@@ -274,6 +276,15 @@ extension SourceKitLSPServer {
     _ req: DocumentTestsRequest,
     workspace: Workspace,
     languageService: LanguageService
+  ) async throws -> [TestItem] {
+    return try await getDocumentTests(req, workspace: workspace, languageService: languageService)
+      .mergingTestsInExtensions()
+  }
+
+  private func getDocumentTests(
```

Comment:
Generally, methods shouldn’t be prefixed with `get` in Swift. 

I think a better name might be `documentTestsWithoutMergingExtensions`